### DISCUSSION TITLE
Openzeppelin-solidity not found

### DISCUSSION THREAD

**Discussion by jsim77** [remix-run/remix#3282] [Q&A] [answered]
Dear Support,

I have a hard time when comes to deploying one of the .sol files into Remix IDE, especially when comes to Import "Openzeppelin-solidity/contracts/.....sol";

I did go to the link and yes, they have changed it to --> https://github.com/OpenZeppelin/openzeppelin-contracts and No More using Openzeppelin-solidity. Problem is, I've changed it as stated directly to the .sol file in Remix IDE but seems that Remix IDE is unable to "accept the change I did" and still pop-up the same "old" error of "Not Found of OpenZeppelin-solidity/contracts/..".

I've searched using Stack Exchange for ways to overcome this, such as using "https://github.com/OpenZeppelin", @OpenZeppelin, and using v2.4.0 or v2.5.0 --> still does not solve this problem.

Sorry, I'm out of ideas for this issue. Any suggestions? I'm using Remix-IDE in Chrome as it's convenient to connect to my MetaMask.

Looking forward to your advice on how to resolve this, thanks.

**Comment by kiliman** [comment, marked as answer]
Hi. This is the repository for the Remix web framework, not Remix IDE.

I believe you want https://github.com/ethereum/remix-project

**Reply by jsim77** [reply]
Hi, noted with thanks.
